applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -71,14 +75,49 @@ namespace Kratos
     //************************************************************************************
     //************************************************************************************
 
-    void BaseSolidElement::FinalizeSolutionStep( ProcessInfo& CurrentProcessInfo )
+    void BaseSolidElement::FinalizeSolutionStep( ProcessInfo& rCurrentProcessInfo )
```

Comment:
It is with the r:

![imagen](https://user-images.githubusercontent.com/19991680/27984851-931e240c-63e0-11e7-93c1-acf705afc31b.png)
 From the element.h